src/Validator/JSON.php

```diff
@@ -24,6 +24,30 @@ public function getDescription()
         return 'Value must be a valid JSON string';
     }
 
+    /**
+     * Is array
+     *
+     * Function will return true if object is array.
+     *
+     * @return bool
+     */
+    public function isArray(): bool
+    {
+        return true;
```

Comment:
@kodumbeats, @eldadfux  for JSON, is it more correct to say 

```
isArray -> false 
getType -> object 
```

Reply:
Agree.